ingest/ingest.go

```diff
@@ -146,6 +147,11 @@ func (i *FileIngest) InsertData(c *fiber.Ctx) error {
 	if !ok {
 		writer = NewFileWriter(
 			dir,
+			i.Config.Ingest.MaxAgeSeconds,
```

Comment:
There's no need to add these additional parameters as they are not related to file compression.